easybuild/easyconfigs/c/CP2K/CP2K-2023.1-foss-2022b.eb

```diff
@@ -0,0 +1,40 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'CP2K'
+version = '2023.1'
+
+homepage = 'https://www.cp2k.org/'
+description = """CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
+ simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different
+ methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and
+ classical pair and many-body potentials. """
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+source_urls = ['https://github.com/cp2k/cp2k/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['dff343b4a80c3a79363b805429bdb3320d3e1db48e0ff7d20a3dfd1c946a51ce']
+
+builddependencies = [
+    ('flex', '2.6.4'),
+    ('Bison', '3.8.2'),
+]
+
+dependencies = [
+    ('Libint', '2.7.2', '-lmax-6-cp2k'),
+    ('libxc', '6.1.0'),
+    ('libxsmm', '1.17'),
+    ('libvori', '220621'),
+    ('FFTW', '3.3.10'),
```

Comment:
This is already part of the toolchain, so could be left out, can't it?

I also see some new dependencies compared to `CP2K-9.1-foss-2022a.eb`. Just for future reference: are these optional and do they enable additional functionality? Or are they simply new (required) dependencies for this newer version of CP2K?

Reply:
> FFTW is there as dependency to avoid that we forget including it when building CP2K with intel toolchain.

Ah yeah, why not. You're right, it doesn't hurt.

> The additional dependencies are either required for CP2K > 9.1, or optional (see https://github.com/easybuilders/easybuild-easyconfigs/pull/18379)

Ok